test/backends/fs_mac_test.exs

```diff
@@ -0,0 +1,9 @@
+defmodule FileSystem.Backends.FSMacTest do
+  use ExUnit.Case
+  import FileSystem.Backends.FSMac
+
```

Comment:
I haven't think about the unit test carefully, but I agree we don't need to run unit test for a backend if the backend doesn't support current os.
the travis-ci only support  `macos` and `linux`, so I think we have to test `windows` and `freebsd` manually.